### ISSUE TITLE
make_2d_deg_day_ts – statistic = mean

### ISSUE THREAD

**Issue by stephanie-owen** [NEFSC/READ_EDAB_Utilities#5]
In make_2d_deg_day_ts, lines 27-43: 

`  data.summary = EDABUtilities::make_2d_summary_ts(data.in = data.in,
                                    write.out = F,
                                    shp.file = shp.file,
                                    var.name = var.name,
                                    agg.time = 'days',
                                    statistic = 'mean',
                                    area.names =area.names)`

Should statistic = 'mean' be either 'nd' or 'dd' (depending on which the user chooses?).

Output table looks like this. I ran this function with statistic = 'nd' but statistic column reads mean. 
![edab_utils_example](https://github.com/user-attachments/assets/f0f0c817-96b6-4177-aa82-71516f41c0e5)

**Comment by jcaracappa1** [comment]
That step makes sure that you're looking at daily means of your variable before you do any dd or nd calculations. This isn't an issue for most products, but it would be if the source product ran on an hourly timestep.

**Comment by jcaracappa1** [comment]
@stephanie-owen I think I see the issue here you should be using make_2d_deg_day_ts() if you want to do that calculation. make_2d_summary_ts() doesn't do the degree day-like operations.

**Comment by stephanie-owen** [comment]
Yes, sorry you are correct. Re-ran this code with the correct function. Marking as closed.
